_episodes/05-join.md

```diff
@@ -464,8 +464,8 @@ As a reminder, here are the columns we want from the Gaia table:
 ~~~
 columns = 'source_id, ra, dec, pmra, pmdec'
 
-query = query_base.format(columns=columns)
-print(query)
+cone_query = cone_base_query.format(columns=columns)
```

Comment:
Do you think this should have a different name since we use `cone_query` in episode 2?

Reply:
Since it is a call back to that episode, I had decided to keep it the same. But I am open to hear an alternative reason.